docs/quickstart.rst

```diff
@@ -76,6 +76,10 @@ We can create some example news files to demonstrate::
     $ echo 'The final part is ignored, so set it to whatever you want.' > src/myproject/newsfragments/8765.removal.txt
     $ echo 'misc is special, and does not put the contents of the file in the newsfile.' > src/myproject/newsfragments/1.misc
 
+If the news fragment doesn't need to be linked to any ticket ID or other identifier, start the file name with ``+``::
```

Comment:
Maybe 

```suggestion
If the news fragment doesn't need to be linked to any ticket ID or other identifier, start the file name with ``+``.
The content will still be included in the final release notes,
in the category corresponding to the file extension::
```

Reply:
Maybe called them "orphan" news fragments in the docs?

Not bug deal, but might help in the future with bug reporting and help us have simple word to describe this feature.

Reply:
I've consistently implemented the wording of "orphan news fragments" across the changes now.